pkgs/development/python-modules/3to2/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchurl
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "3to2";
+  version = "1.1.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
```

Comment:
@FRidh Or just allow to override the default from `tar.gz` to `zip` by the package author. This is how it is done in OpenEmbedded.